README.md

```diff
@@ -1,5 +1,19 @@
-# DNNCommunityTheme
-This repository contains source code for the theme used on [dnncommunity.org](http://dnncommunity.org).
+# DNNCMS DNN Community Site Theme
+## Bootstrap 5 
+
+![DNNCMS Community Site Theme Preview](default.jpg)
+
+## About
+New (2023) DNN community site Theme.
+
+
+## Notes:
+
+- The Theme expects to be in a folder named "DNN-Community-2024" (same for the containers).
+- The Theme uses Gulp to parse the SCSS.
+- Containers are "on change" copied from the  "_src/Containers" folder to the appropriate DNN container folder.
+- The Gulp script assumes the name of your container folder is the same as the skin folder.
+- The MS build script currently assumes you have the community build tasks installed en uses 7 Zip.
```

Comment:
I am not seeing any MS Build files in this theme.  Is the intent to use MS Build to build the theme package?

Reply:
yes

Reply:
I added the build files